widgets/partials/admin-monk-language-switcher.php

```diff
@@ -0,0 +1,27 @@
+<?php
+/**
+ * Monk Language Switcher.
+ *
+ * @link       https://github.com/brenoalvs/monk
+ * @since      1.0.0
+ *
+ * @package    Monk
+ * @subpackage Monk/Widgets/Partials
+ */
+
+$title = ! empty( $instance['title'] ) ? $instance['title'] : __( 'New title', 'text_domain' );
+?>
+<p class="monk-widgets-form">
+	<label for="<?php echo esc_attr( $this->get_field_id( 'title' ) ); ?>"><?php _e( esc_attr( 'Title: ' ) ); ?></label> 
+	<input class="widefat" id="<?php echo esc_attr( $this->get_field_id( 'title' ) ); ?>" name="<?php echo esc_attr( $this->get_field_name( 'title' ) ); ?>" type="text" value="<?php echo esc_attr( $title ); ?>">
+
+	<label><?php _e( esc_attr( 'Show Flags ' ) ); ?></label>
```

Comment:
Join the `_e` and `esc_attr` functions.
WordPress provides an `esc_attr_e` function to make this, don't forget to set the ext-domain value to `monk`